src/components/views/DownloadModModal.vue

```diff
@@ -214,7 +213,10 @@ interface DownloadProgress {
                         downloadVersion: [currentAssignId, obj]
                     });
                 }
-            }, this.downloadCompletedCallback);
+            }, (downloadedMods) => {
+                this.downloadCompletedCallback(downloadedMods);
+                this.downloadingMod = false;
```

Comment:
Just mentioning this in the hopes we remember to do this in the future PRs: this should be something like `isDownloadProgressModalOpen` and probably to be handled into `ModalModule` store. Ideally in a way that when one of the three modals in this file is opened, the other two are automatically closed.